src/Axis.js

```diff
@@ -305,7 +306,10 @@ export default class Axis extends BaseClass {
         if (t !== "1") n = `${t} x ${n}`;
         return d < 0 ? `-${n}` : n;
       }
-      return this._d3Scale.tickFormat ? this._d3Scale.tickFormat(labels.length - 1)(d) : d;
+
+      let n = this._d3Scale.tickFormat ? this._d3Scale.tickFormat(labels.length - 1)(d) : d;
+      n = n.replace(/[^\d\.\-\+]/g, "") * 1;
+      return isNaN(n) ? n : formatAbbreviate(n);
```

Comment:
I just tested this with a time scale, and the formatting is not working. After the "log" if statement, add an else if for "time" scales, and `return this._d3Scale.tickFormat(labels.length - 1)(d);`. Here's my test code:

```js
new d3plus.AxisBottom()
  .domain([2013, 2014])
  .scale("time")
  .render();
```

Reply:
if we convert `d` to string, in the next line we return to number:

`n = n.replace(/[^\d\.\-\+]/g, "") * 1;`

I suggest to add an `else if ` in the statement. I tested and works. If you believe that's a bad idea, only tell me @davelandry :)

```
else if (this._scale === "ordinal") {
    return d;
}
```

Reply:
@cnavarreteliz that solution works for me!